catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.xml

```diff
@@ -281,6 +281,13 @@
             type="Integer"
             default="1048576"/>
 
+        <!--TODO: set default target config-->
+        <AD id="maximumNumListItems"
+            name="Maximum Number of List Items"
+            description="The maximum size (in bytes) to allow per client when receiving a POST/PATCH/PUT. Note: This does not affect product upload size, just the maximum size allowed for calls from Intrigue."
+            type="Integer"
+            default="3"/>
```

Comment:
What should this default be. Is 25 or 100 a good default?

Reply:
250 might be a good number, since we default the page size to that